R/preprocessing.R

```diff
@@ -0,0 +1,152 @@
+##' Fills missing data in a data set to prepare it for use within the package
+##'
+##' @description `r lifecycle::badge("experimental")`
+##'
+##' @param data Data frame with a `date` column. The other columns depend on the
+##'   model that the data are to be used, e.g. [estimate_infections()] or
+##'   [estimate_secondary()]. See the documentation there for the expected
+##'   format.
+##' @param missing Character. Options are "missing" (the default), "accumulate"
+##'   and "zero". This determines how missing dates in the data are interpreted.
```

Comment:
```suggestion
##'   and "zero". This determines how implicit missingness is treated, i.e., how missing dates in the data are interpreted.
```

Reply:
I think if doing this we'd have to define implicit vs. explicit missingness somewhere - do you think it would make it clearer? I have tried to rephrase without introducing new definitions but please comment if you disagree with the approach.